demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -830,6 +835,13 @@ def validate_layoutscontainer(self, structure_validator, pack_error_ignore_list)
         return layout_validator.is_valid_layout(validate_rn=False, id_set_file=self.id_set_file,
                                                 is_circle=self.is_circle)
 
+    def validate_pre_process_rules(self, structure_validator, pack_error_ignore_list):
+        pre_process_rules_validator = PreProcessRulesValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                           print_as_warnings=self.print_ignored_errors,
+                                           json_file_path=self.json_file_path)
+        return pre_process_rules_validator.is_valid_pre_process_rules(validate_rn=False, id_set_file=self.id_set_file,
+                                                is_circle=self.is_circle)
```

Comment:
should be singular as we validate one file at a time
```suggestion
    def validate_pre_process_rule(self, structure_validator, pack_error_ignore_list):
        pre_process_rule_validator = PreProcessRuleValidator(structure_validator, ignored_errors=pack_error_ignore_list,
                                           print_as_warnings=self.print_ignored_errors,
                                           json_file_path=self.json_file_path)
        return pre_process_rule_validator.is_valid_pre_process_rule(validate_rn=False, id_set_file=self.id_set_file,
                                                is_circle=self.is_circle)
```

Reply:
https://github.com/demisto/demisto-sdk/commit/040e4eeed07bbf9c51dc0d67184bd85ea2ed0d5c